diff --git a/.gn b/.gn
index 3b92d0a..a73d00e 100644
--- a/.gn
+++ b/.gn
@@ -55,7 +55,7 @@ default_args = {
   ios_deployment_target = "14.0"
 
   # The SDK API level, in contrast, is set by build/android/AndroidManifest.xml.
-  android_ndk_api_level = 23
+  android_ndk_api_level = 21
 
   # WebRTC does not provide the gflags dependency. Because libyuv uses it only
   # for unittests, it can be disabled (see third_party/libyuv/BUILD.gn)
@@ -66,7 +66,7 @@ default_args = {
   gtest_enable_absl_printers = true
 
   # Differently from Chromium, WebRTC still support SDK 21.
-  default_min_sdk_version = 23
+  default_min_sdk_version = 21
 
   # Prevent jsoncpp to pass -Wno-deprecated-declarations to users
   jsoncpp_no_deprecated_declarations = false
diff --git a/rtc_tools/network_tester/BUILD.gn b/rtc_tools/network_tester/BUILD.gn
index fd98be7..9cdfbf1 100644
--- a/rtc_tools/network_tester/BUILD.gn
+++ b/rtc_tools/network_tester/BUILD.gn
@@ -128,7 +128,7 @@ if (is_android) {
     testonly = true
     apk_name = "NetworkTesterMobile"
     android_manifest = "androidapp/AndroidManifest.xml"
-    min_sdk_version = 23
+    min_sdk_version = 21
     target_sdk_version = 24
 
     deps = [
diff --git a/sdk/android/BUILD.gn b/sdk/android/BUILD.gn
index 5590087..f8765b9 100644
--- a/sdk/android/BUILD.gn
+++ b/sdk/android/BUILD.gn
@@ -1627,7 +1627,7 @@ if (is_android) {
     rtc_instrumentation_test_apk("android_instrumentation_test_apk") {
       apk_name = "android_instrumentation_test_apk"
       android_manifest = "instrumentationtests/AndroidManifest.xml"
-      min_sdk_version = 23
+      min_sdk_version = 21
       target_sdk_version = 24
 
       sources = [
diff --git a/webrtc.gni b/webrtc.gni
index 58df199..9f02ad2 100644
--- a/webrtc.gni
+++ b/webrtc.gni
@@ -525,7 +525,7 @@ template("rtc_test") {
       use_default_launcher = false
       android_manifest = webrtc_root + "test/android/AndroidManifest.xml"
       use_raw_android_executable = false
-      min_sdk_version = 23
+      min_sdk_version = 21
       target_sdk_version = 24
       deps += [
         "//build/android/gtest_apk:native_test_instrumentation_test_runner_java",
diff --git a/build/config/android/config.gni b/build/config/android/config.gni
index 516462806..717d951af 100644
--- a/build/config/android/config.gni
+++ b/build/config/android/config.gni
@@ -73,7 +73,7 @@ if (is_android) {
     if (is_cronet_build) {
       # Cronet's min sdk version support policy is different from the rest of
       # Chromium; Cronet explicitly supports older devices.
-      default_min_sdk_version = 23
+      default_min_sdk_version = 21
     }
 
     # Static analysis can be either "on" or "off" or "build_server". This
@@ -131,7 +131,7 @@ if (is_android) {
   }
 
   # Our build system no longer supports legacy multidex.
-  min_supported_sdk_version = 23
+  min_supported_sdk_version = 21
 
   # ASAN requires O MR1.
   # https://github.com/google/sanitizers/wiki/AddressSanitizerOnAndroid/01f8df1ac1a447a8475cdfcb03e8b13140042dbd#running-with-wrapsh-recommended
